applications/plugins/SofaMatrix/src/SofaMatrix/FillReducingOrdering.inl

```diff
@@ -23,13 +23,19 @@
 
 #include <SofaMatrix/FillReducingOrdering.h>
 
+#include <Eigen/Sparse>
```

Comment:
Eigen dependency should be visible in CMakeLists.txt and Config.cmake.in

Reply:
Done, even if Eigen was already a dependency of another module.

How can I test a Config.cmake.in script?

Reply:
We don't have such test in our CI. 
Currently the best way is to generate binaries with the command `[ci-build][generate-binaries]` and then try to build a plugin on top of those binaries.